src/Microsoft.Extensions.Localization/ResourceManagerStringLocalizer.cs

```diff
@@ -50,7 +56,7 @@ public class ResourceManagerStringLocalizer : IStringLocalizer
         /// </summary>
         public ResourceManagerStringLocalizer(
             ResourceManager resourceManager,
-            AssemblyWrapper resourceAssemblyWrapper,
+            IResourceStringProvider resourceStringProvider,
```

Comment:
A constructor that uses `.Internal` types should be considered internal. @Eilon was happy with that principle when we discussed it.